packages/mdc-list/_mixins.scss

```diff
@@ -21,12 +21,278 @@
+    @include mdc-list-group-subheader-ink-color(text-primary-on-background);
+  }
+
+  .mdc-list--dense {
+    @include mdc-feature-targets($feat-structure) {
+      padding-top: 4px;
+      padding-bottom: 4px;
+      font-size: .812rem;
+    }
+  }
+
+  .mdc-list-item {
+    @include mdc-feature-targets($feat-structure) {
+      @include mdc-list-item-base_;
+    }
+  }
+
+  // "Selected" is ephemeral and likely to change soon. E.g., selecting one or more photos to share in Google Photos.
+  // "Activated" is more permanent. E.g., the currently highlighted navigation destination in a drawer.
+  .mdc-list-item--selected,
+  .mdc-list-item--activated {
+    @include mdc-feature-targets($feat-color) {
+      @include mdc-list-item-primary-text-ink-color(primary);
+      @include mdc-list-item-graphic-ink-color(primary);
+    }
+  }
+
+  .mdc-list-item--disabled {
+    @include mdc-feature-targets($feat-color) {
+      @include mdc-list-item-primary-text-ink-color(text-disabled-on-background);
+    }
+  }
+
+  .mdc-list-item__graphic {
+    @include mdc-feature-targets($feat-structure) {
+      @include mdc-list-graphic-size_(24px);
+
+      flex-shrink: 0;
+      align-items: center;
+      justify-content: center;
+      fill: currentColor;
+    }
+  }
+
+  // Extra specificity is to override .material-icons display style if used in
+  // conjunction with mdc-list-item__graphic
+  // stylelint-disable plugin/selector-bem-pattern
+  .mdc-list .mdc-list-item__graphic {
+    @include mdc-feature-targets($feat-structure) {
+      display: inline-flex;
+    }
+  }
+  // stylelint-enable plugin/selector-bem-pattern
+
+  .mdc-list-item__meta {
+    @include mdc-feature-targets($feat-structure) {
+      @include mdc-rtl-reflexive-property(margin, auto, 0, ".mdc-list-item");
+    }
+  }
+
+  .mdc-list-item__text {
+    @include mdc-feature-targets($feat-structure) {
+      @include mdc-typography-overflow-ellipsis;
+    }
+  }
+
+  // Disable interaction on label elements that may automatically
+  // toggle corresponding checkbox / radio input.
+  .mdc-list-item__text[for] {
+    @include mdc-feature-targets($feat-structure) {
+      pointer-events: none;
+    }
+  }
+
+  .mdc-list-item__primary-text {
+    @include mdc-feature-targets($feat-structure) {
+      @include mdc-typography-overflow-ellipsis;
+      @include mdc-typography-baseline-top($item-primary-text-baseline-height);
+      @include mdc-typography-baseline-bottom($item-secondary-text-baseline-height);
+
+      display: block;
+
+      // stylelint-disable plugin/selector-bem-pattern
+      .mdc-list--dense & {
+        @include mdc-typography-baseline-top($dense-item-primary-text-baseline-height);
+        @include mdc-typography-baseline-bottom($item-secondary-text-baseline-height);
+      }
+      // stylelint-enable plugin/selector-bem-pattern
+    }
+  }
+
+  .mdc-list-item__secondary-text {
+    @include mdc-feature-targets($feat-structure) {
+      @include mdc-typography(body2);
```

Comment:
Wouldn't `mdc-typography(...)` target the `typography` feature?

@mmalerba I assume that other `mdc-typography` mixins can be included under structure and that it's only important for styles pertaining to typographical scale levels to be under the typography feature?

Reply:
I think we should do it the same as theme, styles that depend on the typography global variables that users can fiddle with should be targeted at the typography feature (so in this case, yeah I would think this should be targeted at typography).  If the other typography mixins don't depend on the global variables then I would say, yes we can put them under structure